File: cpp/control.cpp
#include <iostream>

int main() {

    int a = 10;

    if (a % 2 == 0) {
        std::cout << "even" << std::endl;
    } else {
        std::cout << "odd" << std::endl;
    }

    switch (a % 2 ) {
        case 0:
            std::cout << "even number" << std::endl;
            break;
        case 1:
            std::cout << "odd number" << std::endl;
            break;
    }

    int i;
    for (i = 1; i <= 9; i++) {
        std::cout << a << " * " << i << " = ";
        std::cout << a * i << std::endl;
    }

    int j = 1;
    while (j <= 9) {
        std::cout << a << " x " << j << " = ";
        std::cout << a * j << std::endl;
        j++;
    }

    int k;
    for (k = 0; k < 10; k++) {
        if (k % 3 == 0) {
            continue;
        }
        std::cout << k << std::endl;
    }
    
    

}

File: py/control.py
a = 10

if a % 2 == 0:
    print("even")
else:
    print("odd")

a = 4
for i in range(1, 10):
    print(a, "*", i, "=", a * i)

i = 1
a = 5
while i <= 9:
    print(a, "*", i, "=", a * i)
    i += 1

i = 0
while True:
    print(i)
    i += 1
    if i >= 5:
        break

for i in range(10, 20):
    if i % 3 == 0:
        continue
    print(i)

